removeDuplicates.py

```diff
@@ -0,0 +1,41 @@
+# Time Complexity : One: O(n^2) | Two: O(1)
+# Space Complexity : One: O(n) | Two: O(1)
```

Comment:
Time complexity of your second solution will be O(n)

Reply:
Oh yes! By mistake I wrote 1